### DISCUSSION TITLE
SSG with i18n routing causing console error about data/../{locale}/index.json

### DISCUSSION THREAD

**Discussion by so99ynoodles** [vercel/next.js#20966] [Help] [answered]
I'm using SSG with i18n routing, and it works fine.
But a console error annoying me.
![image](https://user-images.githubusercontent.com/46433895/104151071-59aa8980-541f-11eb-91a9-23505831bf17.png)

I visited the source/output page, and it seems like there is no `ja/index.json` file but `ja.json` file.
![image](https://user-images.githubusercontent.com/46433895/104151041-439cc900-541f-11eb-8868-d4d9cdf534cf.png)

I think this is a bug, but if someone knows how to fix this, please let me know.

**Comment by ijjk** [comment, marked as answer]
Hi, are you on the latest version of Next.js `v10.0.5`, some preload cases with i18n have been fixed in the latest version.

**Reply by so99ynoodles** [reply]
Thank you! It solved my problem! 😄
My version was `v10.0.0`
